### ISSUE TITLE
 Cant play vertical videos - transcoding issue ? intel gpu drivers + ffmpeg 4 RESOLVED

### ISSUE THREAD

**Issue by brzeziniol** [pulsejet/memories#1035]
Hello everyone
I know that a video recorded vertically is silly but I can't seem to change my children's habits :)
To the point:
I installed memories
(absolutely great app by the way !!)
Thanks to the simple but very detailed documentation I was able to start transcoding ! and I was already about to switch from photos to memories when I realized, unfortunately, videos recorded vertically do not play (both in the web app and android) 
Horizontal videos work like a champ
I know the problem is unusual but I totally don't know where to start .
Have any of you encountered this before ?
Nextcloud log displays the following error

{"reqId":"ruA2zwgiN4RE9omcoGfE","level":3,"time":"2024-02-07T22:34:26+00:00","remoteAddr":"109.173.224. 169","user":"szymon","app":"memories","method":"GET","url":"/nextcloud/apps/memories/api/video/transcode/56k9ol0osl90/1004857/480p-000000. ts", "message": "Transcode failed: Transcoder returned 408", "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0", "version": "28.0.2.5", "data":{"app": "memories"}, "id": "65c40c6959475"}

Thanks in advance for your help!

**To Reproduce**
Go to memories, choose vertical video, try to play

**Screenshots**
sample video here
https://szymok.pl/nextcloud/s/NLRNiorb5be39GQ

**Platform:**
Client:
- Fedora Linux 39
- Firefox 122

Server:
Debian 11
Intel(R) Pentium(R) Gold G6400 CPU @ 4.00GHz
8 GB RAM
- Memories Version: 6.2.2
- Nextcloud version": "28.0.2.5 installed through yunohost
- PHP Version: 8.2

**Comment by brzeziniol** [comment]
Following basic steps on https://memories.gallery/hw-transcoding/#basic-steps I launched the script (Just had to modify it with apt-get install -y --allow-remove-essential \ instead of apt-get install -y ) 
After that i see that admin panel couldnt find ffmpeg and ffprobe in /usr/bin/ffmpeg / /usr/bin/ffprobe should i change it ?

**Comment by Vessel40** [comment]
You must be having some issue with your ffmpeg install, I can play vertical videos fine. 
Memories Version: 6.2.2
Nextcloud version": "28.0.2" running on Unraid - kilrah/nextcloud-ffmpeg docker.
![Screenshot 2024-02-07 213124](https://github.com/pulsejet/memories/assets/156139342/8cd4f063-4383-47d0-90e0-33a4ddb6cdf5)

**Comment by brzeziniol** [comment]
OK, so I moved forward / problem partially solved

The main problem was caused by ffmpeg v4 after launching script from here https://github.com/pulsejet/memories/blob/master/go-vod/build-ffmpeg-nvidia.sh (modified it by adding  --allow-remove-essential to apt-get install -y)
and symlinked ffmpeg and ffprobe to new location with
sudo ln -s /usr/local/bin/ffmpeg /usr/bin/ffmpeg
sudo ln -s /usr/local/bin/ffprobe /usr/bin/ffprobe

Both portrait (vertical) and regular (horizontal) videos play, but in the web application, the vertical videos are reversed. 
Is there any solution for this ?
Thanks in advance

![Zrzut ekranu 2024-02-08 080706](https://github.com/pulsejet/memories/assets/57943767/99ddc17e-4ad8-4cb2-80c9-f3538ee1b973)

BTW. It would be nice if the admin panel could display a warning about an old version of ffmpeg - is there any chance to implement such functionality?
